textproc/heirloom-doctools/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        n-t-roff heirloom-doctools 160308
+categories          textproc
+platforms           darwin
+supported_archs     i386 x86_64
```

Comment:
We don't generally set `supported_arches` like that. Is there a reason you are setting it?

Reply:
I'm not sure what the guidelines on that are tbh but I've always set it.

Reply:
So basically you are saying that this software is not supported on powerpc? If know that for a fact, then keep this line, if not, then it should be removed.